lms/djangoapps/support/static/support/jsx/entitlements/entitlements.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export function EntitlementSupportPage() {
```

Comment:
Nit: Consider instantiating the functional React component using `ES6` syntax and `default export`ing the method at the end of the file.

```javascript
const EntitlementSupportPage = () => (<div>...</div>);

export default EntitlementSupportPage;
```

Reply:
ooooh